Source/GTMSessionFetcherService.h

```diff
@@ -107,6 +107,16 @@ extern NSString *const kGTMSessionFetcherServiceSessionKey;
 // fetchers begin.
 - (void)resetSession;
 
+// Sets the callback queue, specifying that the provided queue is a concurrent queue.
+//
+// When a concurrent queue is explicitly provided via this setter, the the service
+// will wrap the concurrent queue with a new serial queue for each fetcher instance
+// that is created, ensuring all callbacks for a given fetcher instance occur serially.
```

Comment:
Continue the comment here to match the commit one, i.e. - "... this allows each fetcher's callbacks to processes concurrently while ensuring an individual fetcher's callbacks are processed serially/in order."

Reply:
Updated the comment to match the commit message.